.travis.yml

```diff
@@ -14,9 +14,12 @@ env:
   - WP_VERSION=latest WP_MULTISITE=0
   global:
   - WP_TRAVISCI=travis:phpunit
+  - SVN_REPO: https://plugins.svn.wordpress.org/micropub/
+  - secure: "lWqKSX3SPggddb0ass4pmep1GJ3g/l0iTpRTi+q8qaMlD9jFQ7bwMstXQ0QWPtRHqeX2pjbNTTol4Uw4j9djUdQm4XSOM+I7ki2bAOC0yPTy8sVeJ8uBy1cOI2Az2Ni+bYe4vg6ThhF9f9Q1rtb2Xs+suBZYCbUXP7tin+Z6sE3XMtWxmqaIIHOW4FW3z7ha0Dmxqj4fcqgWirEk8rNY55dt1phPfvWUKLbYdANemXnLz9dVf4aEQu/KfKGwlQFfFq4AILrtpeo3BkBIt6ZnU+AZeecyQSB0LEauphrF5fpKfPhsOksLUO/t6SVlWgSJ6K+36axj/OgqD2Ruo+DwYbHQCcVZ+BRJ2ESmch2hM1YZhzBwHkT21cKR8VCZkJkXtBoou2UjBfsw8SMy5M/taJjlJI9SjBl0ZRv/1sviXlEJovHCqg7yy85hX7kGVrULi1m+dDim96x/+YV3O7/VWZRi3mdwQ2lRBqkdpxqLAPnsgvFiNhrDyEKj1gT3G9sVlEpIrg3rw1KZDcP1eOO/DvchSIl3pW2uupN5fmCA+REFXYbLCTioKsfgOcRjPwUx24d4gPsYWhy7bfCOu1Kz9tZSA11Zbz0bVT5fB+awKcKaQkJg2s1JazaSrjBH/HSGBEXXIOXvGS+mUX/I+t2z1NzroHkmqQHWozXRl2Sh6Cs="
```

Comment:
is this a token for something? if so it shouldn't be in git anywhere, right? please revoke it, generate a new one, and put that token somewhere else. 😁

Reply:
https://docs.travis-ci.com/user/encryption-keys/ - Only Travis can decrypt it.

Reply:
oh wow, ok! please add a comment and docs link. thanks!

Reply:
what user have you used for the commit?

Reply:
That would be my credentials right now. But snarfed wants to move to the Indieweb repo, so we should change to them